### PR TITLE
update to use corefx package

### PR DESCRIPTION
**Pull request by anurse**
No more code sharing!!! 🎉 🍰 🎆 👍 

NOTE: This is currently using a version directly off of `dotnet-core`, so we don't want to check it in until https://github.com/aspnet/Universe/pull/789 finishes.

Fixes https://github.com/aspnet/Home/issues/2801
Closes https://github.com/aspnet/Home/issues/2459
Closes https://github.com/aspnet/Home/issues/2465

### CODE REVIEW

File: samples/EchoApp/Properties/launchSettings.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:3159/",
+      "applicationUrl": "http://localhost:56363/",
```
Comment (jkotalik) [inline samples/EchoApp/Properties/launchSettings.json]: Revert?

**Comment by jkotalik** [comment]
App Veyor is failing hard due to port binding issues I believe.

File: src/Microsoft.AspNetCore.WebSockets/Microsoft.AspNetCore.WebSockets.csproj [hunk 2]
```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Http.Extensions" Version="$(MicrosoftAspNetCoreHttpExtensionsPackageVersion)" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="$(MicrosoftExtensionsOptionsPackageVersion)" />
     <PackageReference Include="System.Numerics.Vectors" Version="$(SystemNumericsVectorsPackageVersion)" />
```
Comment (davidfowl) [inline src/Microsoft.AspNetCore.WebSockets/Microsoft.AspNetCore.WebSockets.csproj]: Remove this

**Comment by davidfowl** [comment]
Remove the unused dependency.

**Comment by anurse** [comment]
This PR got kinda messed up trying to rebase on release, the branch is fine but the PR checks aren't running. Will open a new one.

**Pull request closed without merging**